### ISSUE TITLE
Build and Publishing  on File System and TFS Failed .

### ISSUE THREAD

**Issue by saeedtabrizi** [Microsoft/nodejstools#72]
I was tried to publish my project on TFS but i got several errors .
So , when i decided to publish my project on a filesystem path , i got the same errors .
I don't use Microsoft azure to hosting my app , and i want to run node manually on a private server .

I got this error on tfs and filesystem publishing :

```
C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v12.0\Node.js Tools\Microsoft.NodejsTools.targets(176,5): Error : web.config not found in project, to create a project to deploy to Microsoft Azure you must create an Azure Node.js project.
```

Missing documentation in NTVS about publishing for solving this problems .

**Comment by mousetraps** [comment]
Haven't gotten a chance to look into this yet. What I can tell you is that we haven't really focused much on this scenario, so I'm not suprised it doesn't work perfectly. In the meantime, try editing the targets file to remove the web.config dependency or try adding a basic web.config file to just get work around the issue? https://github.com/Microsoft/nodejstools/blob/master/Nodejs/Product/WebRole/web.config

**Comment by pke** [comment]
Ran into the same problem today. The `.pubxml` also marks `<WebPublishMethod>FileSystem</WebPublishMethod>` as non existing in the schema

**Comment by paulvanbrenk** [comment]
This is fixed in release 1.3, and Visual Studio 2017 Update 3
